### PR TITLE
Feature/df 1331 deploy virus scanning service

### PR DESCRIPTION
**Pull request by cinhtau**
Add installation task for clamav on RHEL 7.6
Add single instance for virus scanning service

### CODE REVIEW

**Comment by cinhtau** [comment]
Successfully deployed on test. Outstanding test on training and staging.

File: playbooks/deploy-virus-scanning-service.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+
+- name: Deployment of the Virus Scanning Service app
+
+  hosts: services
```
Comment (birom) [inline playbooks/deploy-virus-scanning-service.yml]: It should be services2. We do not have enough memory on servces.

File: roles/clamav/templates/freshclam.conf.j2 [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+#PrivateMirror mirror1.mynetwork.com
+#PrivateMirror mirror2.mynetwork.com
+
+# Number of database checks per day.
+# Default: 12 (every two hours)
+#Checks 24
+
+# Proxy settings
+# Default: disabled
+HTTPProxyServer l98fppx1.admin.arbeitslosenkasse.ch
```
Comment (birom) [inline roles/clamav/templates/freshclam.conf.j2]: Proxy host is: proxy.admin.ch

File: roles/clamav/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  vars:
+    packages:
+      - clamav
+      - clamav-data
+      - clamav-scanner-systemd
+      - clamav-update
+
+- name: Check if default configuration backup already exists
+  stat:
+    path: /etc/clamd.conf.j2
```
Comment (birom) [inline roles/clamav/tasks/main.yml]: path: /etc/clamd.conf

File: playbooks/deploy-virus-scanning-service.yml (hunk 1 above)
Comment (cinhtau) [reply playbooks/deploy-virus-scanning-service.yml]: Switch to services2

File: roles/clamav/tasks/main.yml (hunk 3 above)
Comment (cinhtau) [reply roles/clamav/tasks/main.yml]: Config filename is /etc/clamd.d/scan.conf

File: roles/clamav/templates/freshclam.conf.j2 (hunk 2 above)
Comment (cinhtau) [reply roles/clamav/templates/freshclam.conf.j2]: changed to proposed value

**Pull request merged**